portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,194 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>My Portfolio</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900" rel="stylesheet">
+  <link href="https://cdn.jsdelivr.net/npm/@mdi/font@4.x/css/materialdesignicons.min.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Material+Icons" rel="stylesheet">
+  <link href="https://cdn.jsdelivr.net/npm/vuetify@2.x/dist/vuetify.min.css" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
+  <script src="https://cdn.jsdelivr.net/npm/vue@2.x/dist/vue.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/vuetify@2.x/dist/vuetify.js"></script>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <div id="index">
+    <v-app>
+      <div class="layout row wrap">
+        <div class="flex text-xs-center xs4 sm3 md3 lg3 mx-auto">
+          <v-card class="mx-auto">
+            <v-img max-width="300" class="mx-auto" src="images/myself.png">
+            </v-img>
+            <v-card-title>Rui Wang</v-card-title>
+            <v-card-subtitle>CS undergraduate @ FDU</v-card-subtitle>
+
+            <v-card-text class="text--primary">
+              <div>
+                <v-icon left>mdi-map-marker</v-icon>
+                Shanghai, China
+              </div>
+              <div>
+                <v-icon left>mdi-git</v-icon>
+                <a href="https://github.com/xyzforever">Github</a>
+              </div>
+              <div>
+                <v-icon left>mdi-email</v-icon>
+                <a href="mailto:abothird19@gmail.com">Email</a>
+              </div>
+            </v-card-text>
+
+          </v-card>
+
+        </div>
+
+        <div class="flex features xs16 offset-xs1 sm8 offset-md0 md8 lg8 mx-auto">
+          <v-card>
+            <v-tabs>
+              <v-tab>
+                Introduction
+              </v-tab>
+              <v-tab>
+                Project
+              </v-tab>
+              <v-tab>
+                Album
+              </v-tab>
+
+              <v-tab-item>
+                <v-card flat>
+                  <v-card-text>
+                    <h3 class="text--primary"> About Me </h3>
```

Comment:
Naming convention: "text-primary"

Reply:
This is text opacity helper classes defined in Vuetify. The "text--primary" means the text has the same opacity as default text.

Reply:
I see, thanks!